docs/services/matterbridge.md

```diff
@@ -0,0 +1,23 @@
+# Matterbridge
+
+[Matterbridge](https://github.com/42wim/matterbridge)
+
+## Configuration
+
+To enable this service, add the following configuration to your `vars.yml` file and re-run the [installation](../installing.md) process:
+
+```yaml
+########################################################################
+#                                                                      #
+# matterbridge                                                         #
+#                                                                      #
+########################################################################
+
+matterbridge_enabled: true
+matterbridge_toml_src_path: "{{ playbook_dir }}/path/to/your/matterbridge.toml.j2"
```

Comment:
It's probably better to have a `matterbridge_configuration_toml` variable.

People can populate it either inline (in `vars.yml`) or using the Ansible [lookup](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_lookups.html) module (for people who wish to store it in a separate file).

Reply:
done :heavy_check_mark: